Tasks/Tumilovich/task1/task1.py

```diff
@@ -0,0 +1,22 @@
+from decimal import *
+Dep = Decimal(input("What's your deposit? \n"))
+Yrs = Decimal(input('How many years? \n'))
+Int = Decimal(input('Annual interest in % \n'))
+Cap = input('You want to add monthly capitalization? \n yes / no \n')
+
+if Cap == 'yes':
+    _SUM2 = Dep * (1 + Int / Decimal('100') / Decimal('12')) ** (Yrs * Decimal('12'))   # Перестраховался и перед всеми цифрами поставил Decimal и ''. может есть проще способ?
```

Comment:
the decimal is not necessary since you already convert inputs to it, and Python will convert all the numbers in an expression automatically